From akpm@osdl.org Fri Sep 23 00:33:19 2005
Message-Id: <200509230656.j8N6u0EO009407@shell0.pdx.osdl.net>
Subject: USB: sisusb warning fix
To: <greg@kroah.com>
From: <akpm@osdl.org>
Date: Thu, 22 Sep 2005 23:55:22 -0700


From: Andrew Morton <akpm@osdl.org>

drivers/usb/misc/sisusbvga/sisusb.c: In function `sisusb_reset_text_mode':
drivers/usb/misc/sisusbvga/sisusb.c:2461: warning: assignment discards qualifiers from pointer target type

Signed-off-by: Andrew Morton <akpm@osdl.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

 drivers/usb/misc/sisusbvga/sisusb.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/usb/misc/sisusbvga/sisusb.c	2005-09-28 14:08:20.000000000 -0700
+++ gregkh-2.6/drivers/usb/misc/sisusbvga/sisusb.c	2005-09-28 15:01:55.000000000 -0700
@@ -2440,7 +2440,7 @@
 sisusb_reset_text_mode(struct sisusb_usb_data *sisusb, int init)
 {
 	int ret = 0, slot = sisusb->font_slot, i;
-	struct font_desc *myfont;
+	const struct font_desc *myfont;
 	u8 *tempbuf;
 	u16 *tempbufb;
 	size_t written;
